medical_lab/models/medical_lab_patient.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalLabPatient(models.Model):
+    _name = 'medical.lab.patient'
+    _description = 'Medical Labs - Patient'
+    _inherits = {'medical.lab': 'lab_id'}
+
+    lab_id = fields.Many2one(
+        string='Lab',
+        comodel_name='medical.lab',
+        required=True,
+        ondelete='cascade',
+    )
+    state = fields.Selection([
```

Comment:
I've been doing the previous migrations with selection fields done as follows to keep in line with other attributes better. Up to you of course, but just pointing out in case of wanting to keep consistency :)
````python
state = fields.Selection(
    selection=[
        ...
    ],
)
````